pkgs/development/haskell-modules/tensorflow-packages.nix

```diff
@@ -0,0 +1,310 @@
+{ fetchFromGitHub }:
+
+self: super:
+let
+  # https://github.com/tensorflow/haskell/pull/247
+  tensorflow-haskell-src = fetchFromGitHub {
+    owner = "yorickvP";
+    repo = "haskell";
+    rev = "0dba27f38c983de52d8160221665bbc513f7313b";
+    sha256 = "1yflcwi8pxraiidjsg07flw9rdplqxr4x75sc88yd9x3zj5hwpa7";
+    fetchSubmodules = true;
+  };
+in
+/*
+(echo {
+  for f in tensorflow{,-core-ops,-logging,-opgen,-ops,-proto,-records{,-conduit},-test}; do
+    echo $f '= super.callPackage ('
+    cabal2nix . --subpath $f
+    echo ') {};'
+  done
+echo }) | sed 's1src = ./.;1src = tensorflow-haskell-src;1' | nixfmt
```

Comment:
It would be great if there was a separate updater script that basically did the above.

A bunch of other places in nixpkgs have an actual updater script that bumps versions and re-runs commands like this.

Having a script like this would make it much easier for other maintainers to keep the tensorflow stuff updated.